### ISSUE TITLE
Console system dark mode.

### ISSUE THREAD

**Issue by benHuntoon** [N1ckP3rsl3y/cs386-a1#2]
The light from the console screen is too bright for night driving, maybe a dark mode would help.

**Comment by N1ckP3rsl3y** [comment]
I don't believe our data shows this is something we might want to add. Or in other words, let them blind them be blind.

**Comment by benHuntoon** [comment]
@epfor3 @N1ckP3rsl3y How do you think we could make an improvement?
